tests/device_tests/test_msg_change_wipe_code_t1.py

```diff
@@ -138,26 +109,18 @@ def test_set_wipe_code_mismatch(client):
     assert client.features.wipe_code_protection is False
 
     # Let's set a new wipe code.
-    ret = client.call_raw(messages.ChangeWipeCode())
-    assert isinstance(ret, messages.ButtonRequest)
-
-    # Confirm intent to set wipe code.
-    client.debug.press_yes()
-    ret = client.call_raw(messages.ButtonAck())
-
-    # Send the new wipe code for the first time.
-    assert isinstance(ret, messages.PinMatrixRequest)
-    wipe_code_encoded = client.debug.encode_pin(WIPE_CODE4)
-    ret = client.call_raw(messages.PinMatrixAck(pin=wipe_code_encoded))
-
-    # Send the new wipe code for the second time, but different.
-    assert isinstance(ret, messages.PinMatrixRequest)
-    wipe_code_encoded = client.debug.encode_pin(WIPE_CODE6)
-    ret = client.call_raw(messages.PinMatrixAck(pin=wipe_code_encoded))
-
-    # The operation should fail, because the wipe codes are different.
-    assert isinstance(ret, messages.Failure)
-    assert ret.code == messages.FailureType.WipeCodeMismatch
+    with client:
+        client.set_pin([WIPE_CODE4, WIPE_CODE6])
```

Comment:
this is analogous to `client.set_passphrase()`, `set_input_flow()`, `set_expected_responses()` etc., WRT the "respond to a PinMatrixRequest" functionality. it can also set a static pin (https://github.com/trezor/trezor-firmware/pull/840/files#diff-795a7774f4ec3dd5511a590ae400559eR103). we could have two different methods for "static pin" and "pin sequence", but ISTM it's more of a concept/documentation issue?

Reply:
As discussed IRL, let's call it `use_pin()`, which will help to avoid confusion + a docstring would be helpful.

Reply:
please review  81a03ed